### PR TITLE
Add wagtail-markdown

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/wagtail-markdown/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "wagtail-markdown" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "89b24b04d874feb97adb0299b5fbc06ea29b5a835ef765b82846575a1d77f63f" %}
+
+
+package:
```
Comment (jakirkham) [inline recipes/wagtail-markdown/meta.yaml]: nit: Would keep one newline between `package` and Jinja.

File: recipes/wagtail-markdown/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  summary: 'Markdown support for Wagtail'
+  description: |
+     Markdown support for Wagtail
+  doc_url: https://github.com/torchbox/wagtail-markdown
+  dev_url: https://github.com/torchbox/wagtail-markdown
+
+extra:
+  recipe-maintainers:
+    - cshaley
+    - sannykr
```
Comment (jakirkham) [inline recipes/wagtail-markdown/meta.yaml]: Guessing this is ok with you @cshaley. 😉

Might be worth adding a comment in issue ( https://github.com/conda-forge/staged-recipes/issues/139 ) noting what @sannykr can add you as a maintainer on generally. 😄

**Comment by jakirkham** [comment]
Thanks for readding this, @sannykr. Have added a few comments above.

File: recipes/wagtail-markdown/meta.yaml (hunk 2 above)
Comment (cshaley) [reply recipes/wagtail-markdown/meta.yaml]: Yes, it's okay, and done.

**Pull request closed without merging**